File: Data Structures & Algorithms/majority-element/submission-1.java
class Solution {
    public int majorityElement(int[] nums) {
        int cc = 0, res = 0;
        for(int num : nums){
            if(cc == 0){ res = num;}
            if(num == res){cc+=1;}
            else {cc-=1;}
        }
        return res;
    }

}

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        set<int>st;
        for(int num : nums){
            st.insert(num);
        }
        int k = 0;
        for(int num : st){
            nums[k++] = num; 
        }
        return k;
    }
};